### ISSUE TITLE
[Feature Request] Add additional exit conditions to recyclingplant

### ISSUE THREAD

**Issue by XargsUK** [bobloy/Fox-V3#192]
**Is your feature request related to a problem? Please describe.**
If a user runs `[p]recyclingplant` and then does not respond, the bot will continuously spam the channel until the user runs `exit` or until they finish the 'work'. Even bot owners cannot stop the cycle, unloading and loading the cog does not break the loop, so a bot restart is required to break out of this. 

**Describe the solution you'd like**
Realistically, more exit conditions should be provided to prevent the loop. Examples of these exit conditions which I'd recommend would be: 

- If user enters anything other than recycle or trash 3 times within the work, the work is cancelled. 
- If the timeout is breached, the work is cancelled. 

I've already made a modification to break from the work if the timeout is breached, as shown below: 

```python3
try:
                answer = await self.bot.wait_for("message", timeout=20, check=check)
            except asyncio.TimeoutError:
                answer = None

            if answer is None:
                await ctx.send(
                    "{}  slacked off at work, so they were sacked with no pay.".format(ctx.author.display_name)
                )
                break
```

**Comment by XargsUK** [comment]
```python
            try:
                answer = await self.bot.wait_for("message", timeout=20, check=check)
            except asyncio.TimeoutError:
                answer = None

            if answer is None:
                if incorrect == 2:
                    await ctx.send("{} slacked off at work, so they were sacked with no pay.".format(ctx.author.display_name))
                    break
                else:
                    await ctx.send("{} is slacking, and if they carry on not working, they'll be fired.".format(ctx.author.display_name))
                    incorrect += 1
```
This works quite well. The 120 timeout set in the repo is really high, so lowering the timeout to something like 10, and if they don't enter something 3x it cancels could work well.

**Comment by bobloy** [comment]
Closed by #193
